### ISSUE TITLE
New Rule: Space after // in line comment

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#1345]
This rule would enforce or disallow a space immediately following the `//` in a line comment. Examples:

``` js
// this has one space
//this doesn't
```

The rule ID should be `spaced-line-comment` with options of "always" or "never". This should be off by default but turned on with "always" for the ESLint project.
